### PR TITLE
V1.2.2

### PR DESCRIPTION
**Pull request by jonhartzler**
fixes to make Hye-jin's life a little easier.

### CODE REVIEW

File: app/assets/javascripts/components/HoneypotImage.jsx [hunk 1]
```diff
@@ -5,11 +5,14 @@ var HoneypotImage = React.createClass({
...
     return {
       cssStyle: {},
+      alt: "",
+      title: "",
     };
   },
 
   render: function() {
-    return (<img src={this.src()} style={this.props.cssStyle} />)
+    console.log(this.props)
```
Comment (jgondron) [inline app/assets/javascripts/components/HoneypotImage.jsx]: :point_up:

File: app/assets/javascripts/components/showcase_editor/Item.jsx [hunk 2]
```diff
@@ -29,14 +29,14 @@ var Item = React.createClass({
     if (honeypot_image == null) {
       return null;
     }
-
+    console.log(this.props.item);
```
Comment (jgondron) [inline app/assets/javascripts/components/showcase_editor/Item.jsx]: :point_up:

**Comment by jgondron** [comment]
:+1: after logs are removed

**Comment by danthewolfe** [comment]
:+1:

**Pull request merged**